pkg/diagnostics/http_tracing.go

```diff
@@ -168,3 +170,38 @@ func tracestateToRequest(sc trace.SpanContext, req *fasthttp.Request) {
 		}
 	}
 }
+
+// GetSpanAttributesMap builds the span trace attributes map for HTTP calls based on given parameters as per open-telemetry specs
+func GetSpanAttributesMapFromHTTP(componentType, componentValue, method, route, uri string, statusCode int) map[string]string {
```

Comment:
nit. We do not need `FromHTTP` suffix  because it is already in http package.

Reply:
we need that as package name is same.